### ISSUE TITLE
<mods:physicalLocation> not crosswalking on deposit as expected

### ISSUE THREAD

**Issue by mrdsaunders** [tomwrobel/ora_data_model#144]
The deposit crosswalk test screen for Pub ID 1012242 is outputting:
```
 <mods:location>
                <mods:physicalLocation>location test artefact</mods:physicalLocation>
</mods:location>
```

However it is not being ingested. The Sword output for that tag is empty:
```
<mods:location/>
```

What am I missing?

**Comment by tomwrobel** [comment]
@mrdsaunders no idea, I'll take a look tomorrow

**Comment by tomwrobel** [comment]
@mrdsaunders note that this is almost certainly what will happen if you try to redeposit 1012237.

The link between Oxris and ORA is broken, as if it never existed, so Oxris creates a new record in ORA:

```
The issue regarding 1012237 | uuid_74db6297-62a1-41c4-9210-3915a0cb90ff on Oxris-QA
The publication 1012237 on Oxris QA appears to have a link to ORA object uuid_74db6297-62a1-41c4-9210-3915a0cb90ff. This is visible on the publication page. At some point, the title and file information have been harvested from ORA.
The record is present in ORA4-QA-SYMP, and is correctly listed in the OAI-PMH response page. Sword requests to the record return valid and correct responses.
However, we cannot harvest the ORA record in Elements. The manual update method (via the Publications page) does not update the record; requesting a full harvest does not update the record.
A full harvest was performed, but the logs do not show any requests for uuid_74db6297-62a1-41c4-9210-3915a0cb90ff or 1012237. No request was queued for these objects. There is no error in the logs.
A search of every ORA/Hyrax Repotool2 log from the last three months reveals no entry for either uuid_74db6297-62a1-41c4-9210-3915a0cb90ff or 1012237 in any log. There are entries for 1012230, 1012231, 1012232, 1012234, 1012235 and 1012239.
The ORA binary file was, for a time, deleted on ORA-SYMP due to an update error. It is possible that Oxris attempted to harvest the record, only to discover that there was no file present, and took some form of action. There is no record of this that I can find. The object has now been fixed and there is now a file attached to the record - although the status of 'complete (file available)' is probably not correct.
The obvious solution is to re-deposit a file and see if that re-establishes the link. However, I don't want to try that in case that transparently fixes the issue (for this record) without us understanding what is happening here. (edited) 

```

**Comment by mrdsaunders** [comment]
Is that last post means for #145 ?

**Comment by tomwrobel** [comment]
yes!

**Comment by tomwrobel** [comment]
Fixed in [feature/ora_customizations 3c46c23] Add physicalLocation to crosswalks

Depoloyed, @mrdsaunders please retest with a new deposit (the values weren't being ingested on deposit or sent on harvest)

**Comment by tobypitts** [comment]
@mrdsaunders @tomwrobel I have retested with Pubs: 1012284/ uuid_0bd2aa29-c829-4cc2-a44f-20300e8fb4af type: artefact 

Location value can be read in Review forms correctly. Modified value can be sent back to Oxris. I'm happy that this is now working
